data/scripting/field_animations.lua

```diff
@@ -74,8 +74,11 @@ end
 --    :arg time: Optional. The time the whole animation will run.
 --               Defaults to 1000 (1 sec)
 --    :type time: :class:`integer`
+--    :arg permanent: Optional. Set to true for permanently hide. Set to false for unexplore.
+--               Defaults to false
```

Comment:
```suggestion
--    :arg permanent: Optional. Set to ``true`` to hide the fields permanently. Set to ``false`` to mark the fields as unexplored.
--                    Defaults to ``false``.
```
same below

and needs a note how this function behaves if a bob currently sees the fields